src/Equinor.ProCoSys.IPO.Command/EventHandlers/PostSaveEvents/IpoCompletedEmailHandler.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
```

Comment:
remove ref to Domain
The IpiCompetedEvent should have bits and pieces needed, not whole Inivitation object